components/base/kinematicbase/execution.go

```diff
@@ -152,11 +153,11 @@ func (ptgk *ptgBaseKinematics) GoToInputs(ctx context.Context, inputSteps ...[]r
 			currentInputs := []referenceframe.Input{
 				step.arcSegment.StartConfiguration[ptgIndex],
 				step.arcSegment.StartConfiguration[trajectoryAlphaWithinPTG],
+				step.arcSegment.StartConfiguration[startDistanceAlongTrajectoryIndex],
 				{step.arcSegment.StartConfiguration[startDistanceAlongTrajectoryIndex].Value + math.Abs(distIncVel)*timeElapsedSeconds},
-				step.arcSegment.StartConfiguration[endDistanceAlongTrajectoryIndex],
```

Comment:
this confuses me, why are we overwriting what the final distance along the traj is?

Reply:
The way I implemented current inputs for PTG bases in the last PR was wrong; this corrects that.

It could be framed as "how much of the arc requested have I executed", or "how much of the arc requested remains".

Currently in `main` is the latter, but with this PR it became clear that was a poor choice and the former is far more useful.

Reply:
You are entirely correct except for one very small thing:
`StartConfiguration: [index, alpha, start, end]`
We actually do
`StartConfiguration: [index, alpha, start, start]`, because that is the inputs at the beginning of the arc.